src/Servers/Kestrel/Transport.MsQuic/src/Internal/MsQuicConstants.cs

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Transport.MsQuic.Internal
+{
+    internal static class MsQuicConstants
+    {
+        private const uint Success = 0;
+
+        private const uint PendingWindows = 0x703E5;
```

Comment:
MsQuicConstants.Windows.Pending?

Reply:
I prefer that.

Reply:
To make sure I understand, you are suggesting:
```C#
internal static class MsQuicConstants
{
    internal static class Windows
    {
        internal static uint Pending = 0x703E5;
    }
}
```

I wouldn't be able to switch on the uints if they were in an inner class as they aren't const.

Reply:
I can make it an enum maybe?

Reply:
Why would moving to a static class mean they can't be `const`? As long as a `const` is non-`public` (`internal` is OK) and we understand the impact of it being `const`, I'm fine with it. It's on public APIs that we have to be extremely careful with `const`.

Reply:
Why aren't they consts?

Reply:
I'm confused about this conversation. @anurse const is fine, it just that I won't be able to access those value inside of the static class MsQuicConstants (it can only be accessed inside of Windows).

So for example, I'm eventually going to want to add static values like we did in Libuv: 
https://github.com/aspnet/AspNetCore/blob/master/src/Servers/Kestrel/Transport.Libuv/src/Internal/LibuvConstants.cs#L12.

By putting these const values inside of a nested static class, I can't access the const values without another wrapper.

I'll push an update and show what I have now.

@Tratcher what are you referring to?

Reply:
Nvrm, ignore everything. I updated the PR now.